test/PDPVerifier.t.sol

```diff
@@ -1529,6 +1499,7 @@ contract PDPListenerIntegrationTest is Test {
     BadListener badListener;
     uint256 constant challengeFinalityDelay = 2;
     bytes empty = new bytes(0);
+    bytes testExtraData = "test extra data";
```

Comment:
I don't understand the point of this field instead of empty -- we're not testing the extra data handling of the listener only its error propagation.